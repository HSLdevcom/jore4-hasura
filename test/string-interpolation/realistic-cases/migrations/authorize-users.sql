CREATE SCHEMA jore3;
GRANT ALL PRIVILEGES ON SCHEMA jore3 TO xxx_db_jore3importer_username_xxx;

CREATE SCHEMA internal_jore3;
GRANT ALL PRIVILEGES ON SCHEMA internal_jore3 TO xxx_db_jore3importer_username_xxx;

GRANT
  USAGE
  ON SCHEMA jore3
  TO
    xxx_hsl_jore4_db_jore3transformer_username_xxx,
    xxx_hsl_jore4_db_mapmatcher_username_xxx;

GRANT
  SELECT, REFERENCES
  ON ALL TABLES
  IN SCHEMA jore3
  TO
    xxx_hsl_jore4_db_jore3transformer_username_xxx,
    xxx_hsl_jore4_db_mapmatcher_username_xxx;

GRANT
  SELECT
  ON ALL SEQUENCES
  IN SCHEMA jore3
  TO
    xxx_hsl_jore4_db_jore3transformer_username_xxx,
    xxx_hsl_jore4_db_mapmatcher_username_xxx;

GRANT
  EXECUTE
  ON ALL ROUTINES
  IN SCHEMA jore3
  TO
    xxx_hsl_jore4_db_jore3transformer_username_xxx,
    xxx_hsl_jore4_db_mapmatcher_username_xxx;
