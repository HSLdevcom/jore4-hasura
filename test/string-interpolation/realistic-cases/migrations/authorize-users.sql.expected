CREATE SCHEMA jore3 AUTHORIZATION jore3importer;
CREATE SCHEMA internal_jore3 AUTHORIZATION jore3importer;

GRANT
  USAGE
  ON SCHEMA jore3
  TO
    jore3transformer,
    mapmatcher;

GRANT
  SELECT, REFERENCES
  ON ALL TABLES
  IN SCHEMA jore3
  TO
    jore3transformer,
    mapmatcher;

GRANT
  SELECT
  ON ALL SEQUENCES
  IN SCHEMA jore3
  TO
    jore3transformer,
    mapmatcher;

GRANT
  EXECUTE
  ON ALL ROUTINES
  IN SCHEMA jore3
  TO
    jore3transformer,
    mapmatcher;
